spec/spec_helper.rb

```diff
@@ -5,9 +5,6 @@
   add_filter '/spec/'
```

Comment:
We still have simplecov installed, right?

Reply:
Yes.